guiEditor/src/tools.ts

```diff
@@ -3,15 +3,15 @@ import { Grid } from "babylonjs-gui/2D/controls/grid";
 import { Vector2 } from "babylonjs/Maths/math";
 
 export class Tools {
-    public static LookForItem(item: any, selectedEntity: any): boolean {
-        if (item === selectedEntity) {
+    public static LookForItem(item: any, selectedEntity: any, firstIteration : boolean = true): boolean {
+        if (!firstIteration && item === selectedEntity) {

```

Comment:
is that the expected default behavior? always ignore the first group of children?

Reply:
Otherwise the selected container will always be open (even if we don't want it to)